Source/Android/app/src/main/res/layout/activity_cheats.xml

```diff
@@ -45,20 +46,22 @@
             android:id="@+id/cheat_list"
             android:name="org.dolphinemu.dolphinemu.features.cheats.ui.CheatListFragment"
             android:layout_width="match_parent"
-            android:layout_height="match_parent" />
+            android:layout_height="match_parent"
+            tools:layout="@layout/fragment_cheat_list" />
 
         <androidx.fragment.app.FragmentContainerView
             android:id="@+id/cheat_details"
             android:name="org.dolphinemu.dolphinemu.features.cheats.ui.CheatDetailsFragment"
             android:layout_width="match_parent"
-            android:layout_height="match_parent" />
+            android:layout_height="match_parent"
+            tools:layout="@layout/fragment_cheat_details" />
 
     </androidx.slidingpanelayout.widget.SlidingPaneLayout>
 
     <View
         android:id="@+id/workaround_view"
         android:layout_width="match_parent"
-        android:layout_height="0dp"
+        android:layout_height="1px"
```

Comment:
Please include a comment like the one you had in the code before. The reason for this `1px` is very non-obvious otherwise.

Reply:
Fixed